src/scripts/show_originals.json

```
@@ -18,6 +18,11 @@
       "label": "Always show reblogs with contributed content",
       "default": false
     },
+    "showReblogsOfNotFollowing": {
+      "type": "checkbox",
+      "label": "Always show reblogs of accounts you don't follow",
```

label nits:
- We typically refer to the user in first-person in preferences, e.g. "Hide my own posts on the dashboard", "Always show my own reblogs"
- Blogs ≠ Accounts

```suggestion
      "label": "Always show reblogs of blogs I don't follow",
```

---

Yeah, I waffled about that. "Reblogs of blogs" sounds funny. But, indeed, more correct!

---

"posts reblogged from blogs I don't follow" would be the more verbose and less funny-sounding option if we care about that...